lib/cli.rb

```diff
@@ -473,6 +473,8 @@ def self.parse_scopes(scope_string)
   command :list do |c|
     c.switch :verbose, :required => false, :negatable => false,
       :desc => "Display additional information about origin of scopes"
+    c.switch :short, :required => false, :negatable => false,
+      :desc => "List only descripton names"
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.